src/index.html

```diff
@@ -1,12 +1,306 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/main.scss"
+    />
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="#moyo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="is-active nav__link"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#LnC"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__pic"></div>
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
```

Comment:
You need to use the semantic tag for text, fix it everywhere
```suggestion
        <h2 class="card__name">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
        
        <p class="card__code">Product code: 195434</p>
```